.github/workflows/cherry-pick-to-frozen.yml

```diff
@@ -0,0 +1,269 @@
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: ${{ needs.cherry-pick.outputs.cherry_pick_pr_number }},
+              milestone: '${{ github.event.pull_request.milestone.number }}'
+            })
+
+      - name: Comment on original PR about cherry-pick success
+        uses: actions/github-script@v7
+        env:
+          CHERRY_PICK_PR_NUMBER: ${{ needs.cherry-pick.outputs.cherry_pick_pr_number }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ADD_CP_TO_TRUNK_NAG: ${{ needs.cherry-pick-to-trunk-check.outputs.add_cp_to_trunk_nag }}
+          NEXT_BRANCH: ${{ needs.prepare.outputs.next_branch }}
+        with:
+          script: |
+            const cherryPickPrNumber = process.env.CHERRY_PICK_PR_NUMBER;
+            const targetBranch = 'trunk';
+            const { owner, repo } = context.repo;
+
+            let cherryPickPr;
+            try {
+              cherryPickPr = (await github.rest.pulls.get({
+                owner,
+                repo,
+                pull_number: parseInt(cherryPickPrNumber)
+              })).data;
+            } catch (e) {
+              core.setFailed("Could not fetch cherry-pick PR details: " + e.message);
+              return;
+            }
+
+            const cherryPickPrUrl = cherryPickPr.html_url;
+            const cherryPickPrTitle = cherryPickPr.title;
+
+            let body = 'IMPORTANT: Merging this PR to the appropriate branches is critical to the release process and ensures that the bug does not cause regressions in the future releases.\n\n' +
+                       `Cherry picking was successful for \`${targetBranch}\`. Please merge the following PR: [${cherryPickPrTitle}](${cherryPickPrUrl})`;
+
+            if (process.env.ADD_CP_TO_TRUNK_NAG === 'true') {
+              body += `\n\nNOTE: This PR might need to be cherry-picked to trunk.`;
+            }
+
+            await github.rest.issues.createComment({
+              owner,
+              repo,
+              issue_number: parseInt('${{ needs.prepare.outputs.pr_number }}'),
+              body
+            });
+
+      - name: Notify Slack on success
+        uses: archive/github-actions-slack@c643e5093620d65506466f2c9b317d5d29a5e517 # v2.10.1
+        env:
+          CHERRY_PICK_PR_URL: https://github.com/${{ github.repository }}/pull/${{ needs.cherry-pick.outputs.cherry_pick_pr_number }}
+          SOURCE_PR_URL: ${{ github.event.pull_request.html_url }}
+          SOURCE_PR_NUMBER: ${{ github.event.pull_request.number }}
+          SOURCE_PR_TITLE: ${{ github.event.pull_request.title }}
+          BASE_BRANCH: ${{ needs.prepare.outputs.base_branch }}
+          ADD_CP_TO_TRUNK_NAG: ${{ needs.cherry-pick-to-trunk-check.outputs.add_cp_to_trunk_nag }}
+          NEXT_BRANCH: ${{ needs.prepare.outputs.next_branch }}
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
```

Comment:
What do you think about adding `slack-optional-unfurl_links: false` to avoid the embedded preview from links? Here and on the other slack notifications of the PR.